src/main/java/frc/robot/commands/OrientedDrive.java

```diff
@@ -48,9 +48,14 @@ public void execute() {
 
         Translation2d robotTranslation = new Translation2d(translationMagnitude, translationAngle);
 
-        double rotation = m_controller.getRightX() * MAX_ROTATION_VELOCITY_RAD;
+        double rotation = 0;
 
-        m_drivetrain.drive(robotTranslation, Rotation2d.fromDegrees(rotation), m_fieldOriented, false);
+        if (m_controller.getXButton()) {
```

Comment:
This is not a good way to turn in place unfortunately. Not enough control. Let's look at this more together and see if we can figure out what's going on.